# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/libraw1394/multi-line-string.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- libraw1394-0.9.0/src/testlibraw.c.orig-rock-maint	2003-08-30 23:28:39.000000000 +0200
+++ libraw1394-0.9.0/src/testlibraw.c	2003-08-30 23:29:38.000000000 +0200
@@ -19,14 +19,14 @@
 
 #define TESTADDR (CSR_REGISTER_BASE + CSR_CYCLE_TIME)
 
-const char not_compatible[] = "\
-This libraw1394 does not work with your version of Linux. You need a different
-version that matches your kernel (see kernel help text for the raw1394 option to
-find out which is the correct version).\n";
-
-const char not_loaded[] = "\
-This probably means that you don't have raw1394 support in the kernel or that
-you haven't loaded the raw1394 module.\n";
+const char not_compatible[] = 
+"This libraw1394 does not work with your version of Linux. You need a different"
+"version that matches your kernel (see kernel help text for the raw1394 option to"
+"find out which is the correct version).\n";
+
+const char not_loaded[] = 
+"This probably means that you don't have raw1394 support in the kernel or that"
+"you haven't loaded the raw1394 module.\n";
 
 quadlet_t buffer;
 
